cmd/config/cache/lookup.go

```diff
@@ -31,16 +31,20 @@ const (
 	Expiry  = "expiry"
 	MaxUse  = "maxuse"
 	Quota   = "quota"
+	MinHits = "minhits"
+
+	EnvCacheDrives  = "MINIO_CACHE_DRIVES"
+	EnvCacheExclude = "MINIO_CACHE_EXCLUDE"
+	EnvCacheExpiry  = "MINIO_CACHE_EXPIRY"
+	EnvCacheMaxUse  = "MINIO_CACHE_MAXUSE"
+	EnvCacheQuota   = "MINIO_CACHE_QUOTA"
+	EnvCacheMinHits = "MINIO_CACHE_MIN_HITS"
```

Comment:
The env value has to match the key name so you can't have MIN_HITS call it MINHITS like MAXUSE